File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1180_basico.js
import { question } from "readline-sync";

function main() {
    const N = get_Number();
    let x = get_Text().split(` `);
    let vetorX = [];
    let menor = Infinity;
    let posicao = 0;

    if (N > 1 && N < 1000) {
        for (let i = 0; i < N; i++) {
            let numero = parseInt(x[i]);
            vetorX.push(numero);

            if (numero < menor) {
                menor = numero;
                posicao = i;

            }
        }

        print(`Menor valor: ${menor}`);
        print(`Posicao: ${posicao}`);

    }

}

function get_Number() {
    return Number(question());
}

function get_Text() {
    return question();
}

function print(texto) {
    return console.log(texto);
}

main();

File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1180_basico.py
def main():
    n = get_Number()
    x = get_Text().split(" ")
    menor = float('inf') 
    posicao_Menor = 0

    for i in range (n):
        numero = int(x[i])

        if (numero < menor):
            menor = numero
            posicao_Menor = i

    print("Menor valor:", menor)
    print("Posicao:", posicao_Menor)

def get_Number():
    return int(input().strip())

def get_Text():
    return input().strip()

main()